node/impl/full/chain.go

```diff
@@ -105,6 +106,10 @@ func (m *ChainModule) ChainGetTipSet(ctx context.Context, key types.TipSetKey) (
 	return m.Chain.LoadTipSet(key)
 }
 
+func (m *ChainModule) ChainGetPath(ctx context.Context, from, to types.TipSetKey) ([]*api.HeadChange, error) {
```

Comment:
change allows `ChainModuleAPI` to satisfy the `EventsAPI` interface: https://github.com/filecoin-project/lily/pull/745/files#diff-a2a1458325b025a5434765ff39f72c2a2035ef2efdb9aacfaddad32505945088L22